DocuSignSamplePlugin/DocuSignPlugin/src/main/java/com/ibm/icn/extension/docusign/service/DocuSignConfiguration.java

```diff
@@ -13,10 +13,7 @@
 
 /*****************************************************************************************************************
  * This class stores the DocuSign credentials required to connect to DocuSign systems.
- * The credentials required to authenticate to a DocuSign system are 
- * //Deprecated.
- * //TODO: Migrate TO OAUTH2.0
- * 				integrator key (application id or client id)  
+ * The credentials required to authenticate to a DocuSign system are
```

Comment:
It seems we can remove this partial sentence.

Reply:
Corrected